ansible/roles/vm_set/library/sonic_kickstart.py

```diff
@@ -108,7 +108,7 @@ def logout(self):
 
 def session(new_params):
     seq = [
-        ('while true; do if [ $(systemctl is-active swss) == "active" ]; then break; fi; echo $(systemctl is-active swss); sleep 1; done', [r'#'], 180),
+        ('while true; do if [ $(systemctl is-system-running) == "degraded" ]; then break; fi; echo $(systemctl is-system-running); sleep 1; done', [r'#'], 180),
```

Comment:
Can you add the check for the state "is-system-running" == "running" also ? It could go into this state as well.
https://www.freedesktop.org/software/systemd/man/systemctl.html.